pagespeed/system/system_rewrite_driver_factory.h

```diff
@@ -417,6 +417,10 @@ class SystemRewriteDriverFactory : public RewriteDriverFactory {
   int num_rewrite_threads_;
   int num_expensive_rewrite_threads_;
 
+  // if true, pagespeed will use envoy fetcher
+  bool use_envoy_fetcher_ = false;
+
+
```

Comment:
nit: superfluous newline

Reply:
Done.
code pushed in https://github.com/apache/incubator-pagespeed-mod/pull/1969/commits/fda843264b794b36ea58e8c160c586a56579eda9